src/lib/viewers/BaseViewer.js

```diff
@@ -588,9 +588,11 @@ class BaseViewer extends EventEmitter {
      * @protected
      * @param {Array} [js] - js assets
      * @param {Array} [css] - css assets
+     * @param {boolean} [isThirdParty] is the asset to load third party
      * @return {Promise} Promise to load scripts
      */
-    loadAssets(js, css) {
+    loadAssets(js, css, isThirdParty = true) {
+        const disableRequireJS = isThirdParty && !!this.options.pauseRequireJS;
```

Comment:
I added the third party boolean because we don't want to emit our message when loading annotations or any other potential code split points. These do not rely on require and should not be considered in this process.

Reply:
`isViewerAsset`? I'm thinking of a future where we have 3rd party assets for annotations or other splits?

Reply:
true but there are a ton of places where we are loading viewer assets, so those would all have to set a boolean

Reply:
and it would also created a flipped negative `!isViewerAsset`